### ISSUE TITLE
Suggestion: Make a note about orientation in the readme file

### ISSUE THREAD

**Issue by AaronGuoZY** [jilei-hao/util#3]
I think it might be helpful to make a note about making sure the two images have the same orientation.

**Comment by jilei-hao** [comment]
That's right. I'll also see if I can add some detection mechanism and a warning message.
